helloworld/src/main/java/FooBar.java

```diff
@@ -1,6 +1,7 @@
 public class FooBar {
 
   public int process(int a) {
+    int b = 0;
     if (a > 1 && a < 10) {
       return a;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'if' child have incorrect indentation level 6, expected level should be 12. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck)